src/inputs/select.scss

```diff
@@ -68,7 +68,7 @@ $iui-after-border: 5px solid;
       }
     }
 
-    &.iui-active {
+    &.iui-expanded {
```

Comment:
Do not like this

Reply:
Don't start refactoring...

Reply:
Reverted class name change and removed addition of `.iui-expanded` from button and header button.